keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -329,6 +330,7 @@ device manager.
 * Creating any more latency than there is today for scheduling:  We should be 
 careful not to add any latency into scheduling over what exists today for default behavior.
 * CPU & Memory Management on scheduler side.
+* Optimal binpacking of resources (we will still consider defragmentation approaches in Beta).
```

Comment:
If you are narrowing down the scope (which I'd recommend given the existing scope), I'd delimit the perimeter in more generic terms. For example, I'd say in non-goals that optimizing the implementation is deferred to beta/post- beta stage.
IMO what we should clarify now at beginning/alpha stage is the roadmap of this work and make sure the initial groundwork allows the planned future extensions and set the directions, I'd postpone details (like IIUC this one) to later stages.
Let's also hear what other reviewers think

Reply:
Added.